src/Commands/Modules/RenameModule.php

```diff
@@ -0,0 +1,376 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ */
+
+namespace FOP\Console\Commands\Modules;
+
+use Composer\Console\Application;
+use FOP\Console\Command;
+use Module;
+use PrestaShop\PrestaShop\Core\Addon\Module\ModuleManagerBuilder;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+final class RenameModule extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('fop:modules:rename')
+            ->setDescription('Rename module')
+            ->setHelp('This command allows you to replace the name of a module in the files and database.')
+            ->addArgument(
+                'old-name',
+                InputArgument::REQUIRED,
+                'Module current name with following format : Prefix_ModuleCurrentNameCamelCased'
```

Comment:
Yes precisely, you have to use the module class name instead of the module name.
I tried to make it clearer in https://github.com/friends-of-presta/fop_console/pull/154/commits/46afc4b8d5df0e4cc0b06a869f24758f3f9cbc5c, even if it's still not ideal. If you have suggestions to make it even clearer, please let me know.